test/OpenTelemetry.Tests/Metrics/MetricAPITest.cs

```diff
@@ -409,18 +384,21 @@ void ProcessExport(Batch<Metric> batch)
         public void TestMetricPointCap(AggregationTemporality temporality)
         {
             var metricItems = new List<Metric>();
-            int metricPointCount = 0;
-            var metricExporter = new TestExporter<Metric>(ProcessExport);
+            var metricExporter = new InMemoryExporter<Metric>(metricItems);
 
-            void ProcessExport(Batch<Metric> batch)
+            int MetricPointCount()
```

Comment:
Thoughts on moving this function to the `InMemoryExporter` class?

Reply:
I would vote for keeping it non-public for now. And if we've found sufficient use cases, make it public later.